File: CPP/Baju.cpp
#include "Aksesoris.cpp"

class Baju : public Aksesoris
{
private:
    string untuk;
    string size;
    string Merek;

public:

    Baju()
    {
    }

    Baju(int id, string namaProduk, int hargaProduk, int stokProduk, string jenis, string bahan, string warna, string untuk, string size, string Merek)
    {
        setId(id);  // Pakai setter
        setNamaProduk(namaProduk);
        setHargaProduk(hargaProduk);
        setStokProduk(stokProduk);
        setJenis(jenis);
        setBahan(bahan);
        setWarna(warna);
        this->untuk = untuk;
        this->size = size;
        this->Merek = Merek;
    }

    string getUntuk()
    {
        return untuk;
    }

    void setUntuk(string untuk)
    {
        this->untuk = untuk;
    }

    string getSize()
    {
        return size;
    }

    void setSize(string size)
    {
        this->size = size;
    }

    string getMerek()
    {
        return Merek;
    }

    void setMerek(string Merek)
    {
        this->Merek = Merek;
    }

    // Destructor
    ~Baju()
    {
    }
    
};

File: PHP/Baju.php
<?php
include 'Aksesoris.php';

class Baju extends Aksesoris
{
    private $untuk;
    private $size;
    private $merek;

    public function __construct($id, $namaProduk, $harga_produk, $stok_produk, $foto_produk, $jenis, $bahan, $warna, $untuk, $size, $merek)
    {
        parent::__construct($id, $namaProduk, $harga_produk, $stok_produk, $foto_produk, $jenis, $bahan, $warna);
        $this->untuk = $untuk;
        $this->size = $size;
        $this->merek = $merek;
    }

    public function setUntuk($untuk)
    {
        $this->untuk = $untuk;
    }

    public function getUntuk()
    {
        return $this->untuk;
    }

    public function setSize($size)
    {
        $this->size = $size;
    }

    public function getSize()
    {
        return $this->size;
    }

    public function setMerek($merek)
    {
        $this->merek = $merek;
    }

    public function getMerek()
    {
        return $this->merek;
    }

    public function addBaju($id, $namaProduk, $harga_produk, $stok_produk, $foto_produk, $jenis, $bahan, $warna, $untuk, $size, $merek)
    {
        parent::setId($id);
        parent::setNamaProduk($namaProduk);
        parent::setHargaProduk($harga_produk);
        parent::setStokProduk($stok_produk);
        parent::setFotoProduk($foto_produk);
        parent::setJenis($jenis);
        parent::setBahan($bahan);
        parent::setWarna($warna);
        $this->untuk = $untuk;
        $this->size = $size;
        $this->merek = $merek;
    }

    public function __destruct()
    {
    }
}